docs/installation.md

```diff
@@ -29,16 +29,18 @@ You will need to create API Keys (Usually you get `key` and `secret`) from the E
 Freqtrade provides a Linux/MacOS script to install all dependencies and help you to configure the bot.
 
 !!! Note
-    Python3.6 or higher and the corresponding pip are assumed to be available. The install-script will warn and stop if that's not the case.
+    Python3.6 or higher and the corresponding `pip` are assumed to be available. The install-script will warn and stop if that's not the case.
 
 ```bash
 git clone git@github.com:freqtrade/freqtrade.git
 cd freqtrade
+git checkout master  # Optional, see (1)
 ./setup.sh --install
 ```
+(1) This command switches the cloned repository to the use of the `master` branch. This command is not needed if you wish to stay on the `develop` branch. You may later switch between branches at any time with the `git checkout master`/`git checkout develop` commands.
```

Comment:
maybe switch the order of the :  `Note "Version considerations"` and the `(1)` parts because this give a technical explanation of what do the command but the user don't know what's behind  master and develop yet ( as the response is below) ? Or up the `Note "Version considerations"` line 33 ?

Reply:
Well, the '(1)' is a comment on the code block (annotation of a line in it) so should preferably be just below the codeblock...
I'll consider moving the note above, to be before the codeblock with commands... 👍